www/docs/client/links/httpBatchStreamLink.md

```diff
@@ -0,0 +1,90 @@
+---
+id: httpBatchStreamLink
+title: HTTP Batch Stream Link
+sidebar_label: HTTP Batch Stream Link
+slug: /links/httpBatchStreamLink
+---
+
+`unstable_httpBatchStreamLink` is a [**terminating link**](./index.md#the-terminating-link) that batches an array of individual tRPC operations into a single HTTP request that's sent to a single tRPC procedure (equivalent to [`httpBatchLink`](./httpBatchLink.md)), but doesn't wait for all the responses of the batch to be ready and streams the responses as soon as any data is available.
+
+:::info
+We have prefixed this as `unstable_` as it's a new API, but you're safe to use it! [Read more](/docs/faq#unstable).
+:::
+
+## Usage
+
+> All usage and options are identical to [`httpBatchLink`](./httpBatchLink.md).
+
+You can import and add the `httpBatchStreamLink` to the `links` array as such:
+
+```ts title="client/index.ts"
+import { createTRPCProxyClient, httpBatchStreamLink } from '@trpc/client';
+import type { AppRouter } from '../server';
+
+const client = createTRPCProxyClient<AppRouter>({
+  links: [
+    httpBatchStreamLink({
+      url: 'http://localhost:3000',
+    }),
+  ],
+});
+```
+
+After that, you can make use of batching by setting all your procedures in a `Promise.all`. The code below will produce exactly **one** HTTP request and on the server exactly **one** database query:
+
+```ts
+const somePosts = await Promise.all([
+  trpc.post.byId.query(1),
+  trpc.post.byId.query(2),
+  trpc.post.byId.query(3),
+]);
+```
+
+## Streaming mode
+
+> ⚠️ This link is unstable and may change in the future.
+
+When batching requests together, the behavior of a regular `httpBatchStreamLink` is to wait for all requests to finish before sending the response. If you want to send responses as soon as they are ready, you can use this link. This is useful for long-running requests.
```

Comment:
This sentence is a description of the differences between `httpBatchStreamLink` and `httpBatchLink`. It is normal to have `httpBatchLink` mentioned in it.

```suggestion
When batching requests together, the behavior of a regular `httpBatchLink` is to wait for all requests to finish before sending the response. If you want to send responses as soon as they are ready, you can use `httpBatchStreamLink` instead. This is useful for long-running requests.
```